llvm/test/CodeGen/NVPTX/bug105730.ll

```diff
@@ -0,0 +1,17 @@
+; RUN: llc < %s -verify-machineinstrs
+
+; Check that llc doesn't crash.
+
+target triple = "nvptx64-nvidia-cuda"
+
+define void @__builtin_splat_i8(i32 %0) {
```

Comment:
Perhaps we could add a more direct MIR test here: llvm/test/CodeGen/NVPTX/proxy-reg-erasure-mir.ll

Reply:
Ah thanks!

Reply:
done. PTAL